mlir/test/IR/print-unique-ids.mlir

```diff
@@ -0,0 +1,40 @@
+// RUN: mlir-opt -mlir-print-unique-ids %s | FileCheck %s
+
+// CHECK: %arg5
+// CHECK: %15
+module {
+  func.func @uniqueConflicts(%arg0 : memref<i32>, %arg1 : memref<i32>) {
```

Comment:
This test seems unnecessarily large. Don't you only need to test a function with two `scf.for` ops?

Reply:
I suppose I wanted to make sure it worked for nesting depths > 1, but I can reduce.